doc/dev/develop.md

```diff
@@ -0,0 +1,45 @@
+## Development
+
+You can install RISE in development mode in this way:
+
+### Install npm
+
+First, you'll need to install `npm` (and `node`, `conda install nodejs` is a good idea).
+
+**Step 1.** Install the JS dependencies:
+
+    npm install
+
+**Step 2.** Copy reveal into the static folder and avoid reveal.js RESET styling:
+
+    npm run build-reveal
+    npm run reset-reveal
+
+To remove `reveal.js` from the static folder you can use `npm run clean-reveal`.
+
+**Step 3.** Build the CSS assets:
+
+    npm run build-css
+
+To have per-save automatic building of CSS, you can use::
+
+```
+npm run watch-less
```

Comment:
I would suggest using the same formatting that the other ones above.

Reply:
of course